csharp/ql/src/semmle/code/csharp/frameworks/Dapper.qll

```diff
@@ -0,0 +1,42 @@
+/**
+ * Classes for modelling Dapper.
```

Comment:
```suggestion
 * Classes for modeling Dapper.
```

Reply:
Thanks. We seem to be using both variants, but it would make sense to unify these.